### PR TITLE
OZ-545: Fix version

### PR DESCRIPTION
**Pull request by VaishSiddharth**
JIRA: https://mekomsolutions.atlassian.net/browse/OZ-545

### CODE REVIEW

File: integration-tests/pom.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
   <parent>
     <groupId>com.ozonehis</groupId>
     <artifactId>eip-odoo-openmrs-parent</artifactId>
-    <version>2.0.0-SNAPSHOT</version>
+    <version>2.1.0-SNAPSHOT</version>
```
Comment (corneliouzbett) [inline integration-tests/pom.xml:16]:
```suggestion
    <version>${project.version}</version>
```

File: test-infra-odoo/pom.xml [hunk 2]
```diff
@@ -13,7 +13,7 @@
   <parent>
     <groupId>com.ozonehis</groupId>
     <artifactId>eip-odoo-openmrs-parent</artifactId>
-    <version>2.0.0-SNAPSHOT</version>
+    <version>2.1.0-SNAPSHOT</version>
```
Comment (corneliouzbett) [inline test-infra-odoo/pom.xml:16]:
```suggestion
    <version>${project.version}</version>
```

File: odoo-openmrs/pom.xml [hunk 3]
```diff
@@ -13,7 +13,7 @@
   <parent>
     <groupId>com.ozonehis</groupId>
     <artifactId>eip-odoo-openmrs-parent</artifactId>
-    <version>2.0.0-SNAPSHOT</version>
+    <version>2.1.0-SNAPSHOT</version>
```
Comment (corneliouzbett) [inline odoo-openmrs/pom.xml:16]:
```suggestion
    <version>${project.version}</version>
```

File: integration-tests/pom.xml [hunk 4]
```diff
@@ -30,7 +30,7 @@
     <dependency>
       <groupId>com.ozonehis</groupId>
       <artifactId>eip-odoo-openmrs</artifactId>
-      <version>2.0.0-SNAPSHOT</version>
+      <version>2.1.0-SNAPSHOT</version>
```
Comment (corneliouzbett) [inline integration-tests/pom.xml:33]:
```suggestion
      <version>${project.version}</version>
```

**Review by corneliouzbett — CHANGES_REQUESTED** [review]

File: test-infra-odoo/pom.xml (hunk 2 above)
Comment (VaishSiddharth) [reply test-infra-odoo/pom.xml:16]: Inside parent definition properties can't be added @corneliouzbett 
`Properties in parent definition are prohibited `

File: integration-tests/pom.xml (hunk 1 above)
Comment (VaishSiddharth) [reply integration-tests/pom.xml:16]: Same here
> Inside parent definition properties can't be added @corneliouzbett 
`Properties in parent definition are prohibited `

File: odoo-openmrs/pom.xml (hunk 3 above)
Comment (VaishSiddharth) [reply odoo-openmrs/pom.xml:16]: Same here
> Inside parent definition properties can't be added @corneliouzbett 
`Properties in parent definition are prohibited `

**Pull request merged**